app/assets/javascripts/components/student/student_guardian_edit.es6.jsx

```diff
@@ -86,6 +86,7 @@ class StudentGuardianEdit extends Component {
           content={'Guardian Information'}
           icon={TypeConstants.icons.save} />
         <div style={this.styles.form}>
+          <span>Guardian One</span>
```

Comment:
Let's do this instead to match the `StudentGuardian` component: `<h4>{'Guardian One'}</h4>`.